components/match2/commons/match_group.lua

```diff
@@ -126,14 +149,16 @@ function MatchGroup.MatchByMatchId(args)
 
 	assert(match, 'Match bracketId= ' .. bracketId .. ' matchId=' .. matchId .. ' not found')
 
-	local SingleMatchDisplay = Lua.import('Module:MatchGroup/Display/SingleMatch', {requireDevIfEnabled = true})
-	local config = SingleMatchDisplay.configFromArgs(args)
+	return MatchGroup._displaySingleMatch(args, fullMatchId)
```

Comment:
Hmm I'd say don't extract to a function and instead rename the function this code resides in

Reply:
The function this code resides in is intended to be used for displaying a single match that gets extracted from a bracket or matchlist.
so imo renaming it is not such a good idea
but we could just use it instead
only would have to adjust some stuff for the non main space id prefixing stuff

Reply:
Well in that case referring to a single match here doesn't really make sense, right? Since that's now a type of MatchGroup

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/1154/commits/cea13b0275068cf668cf7a82149745b4e36d8950 adjusts it to use `MatchGroup.MatchByMatchId` inside the new function `MatchGroup.SingleMatch` and hence kicking function `MatchGroup._displaySingleMatch`